sdk/include/reactos/debug.h

```diff
@@ -18,9 +18,6 @@
 #define __RELFILE__ __FILE__
```

Comment:
I modified this header to only use ntdll, and not kernel32.dll so that we can use this in all modules, instead of having 2 or 3 different versions of this file.

Shims and verifier dll's cannot import from kernel32!

What is the reason to add the kernel32 dependency back?

Reply:
Yeah I'm not a huge fan of the idea, either. `ntdll` is always safe to import from, `kernel32` is not. So you're adding extra complexity to the code just to avoid ntdll imports. Because it's "cleaner", by some definition thereof?
Dependencies on ntdll don't hurt because it's always loaded in every process anyway. This fact isn't officially documented for Win32 applications, but (a) it's not something Microsoft can change at this point without breaking a ton of apps, (b) we're writing an OS anyway and a ton of our code relies on undocumented OS internals.

An argument can be made that applications should use kernel32 instead of ntdll, because they make sense on their own, even if you take them away from ROS. But for system DLLs (which seems to be what most of the affected modules here are) I see more potential problems than benefits.

Reply:
`IsDebuggerPresent` and `OutputDebugString` are of `kernel32`.